philly-aging-out/js/map.js

```diff
@@ -7,26 +7,111 @@ var map = L.map('map').setView([39.969, -75.216], 13);
     accessToken: 'pk.eyJ1IjoiZGFuYWVjbW9ibGV5IiwiYSI6ImI4NTk0YWM4MWJlYjJiY2Y5ZWZmNjlmM2ViMGExYzk4In0.Lo6VOcZ5ZkTq9VVYDJo4Ew'
     }).addTo(map);
 
-   //farmer market locations
-    function addMarketToMap(data, map) {
-    var dataLayer = L.geoJson(data);
-    dataLayer.addTo(map);
+    //start Census Tracts
+    function getColor(percentage) {
+        return percentage > 30  ? '#7A0177' :
+               percentage > 20  ? '#C51B8A' :
+               percentage > 12   ? '#F768A1' :
+               percentage > 6   ? '#FBB4B9' :
+                          '#FEEBE2';
+    }
+    function addTractsToMap(data, map) {
+        var layerOptions= { 
+            style: function(featureData) {
+                console.log(featureData.properties.PERCENT65);
+                return {
+                    fillColor: getColor(featureData.properties.PERCENT65),
+                    weight: 1,
+                    opacity: 1,
+                    color: 'white',
+                    fillOpacity: 0.7,
+                };
+            },
+            onEachFeature: function (featureData, layer) {
+            layer.bindPopup("<b>Census Tract" + " " + featureData.properties.NAME10 + "</b></br>" + "<b>Total Population:</b>" + " " + featureData.properties.POPULATION + "</br>" 
+            + "<b>Median Age</b>:" + " " + featureData.properties.MEDIANAGE + "</br>" + "<b>% Over 65</b>:" + " " + featureData.properties.PERCENT65 + "</br>");
+            }
+        };
+        var dataLayer = L.geoJson(data, layerOptions);
+       dataLayer.addTo(map);
```

Comment:
After this line, you can add 

```javascript
dataLayer.bringToBack();
```
to force the census tract layer (`dataLayer`) to be on the bottom stack of layers.